bootstrap/README.md

```diff
@@ -268,13 +268,45 @@ $ cargo run --example upload -- \
 
 Prepare upgrade arguments
 ```shell
+# https://internetcomputer.org/docs/references/bitcoin-how-it-works#api-fees-and-pricing
+$ TESTNET_FEES="record { 
+  get_utxos_base = 20_000_000 : nat;
+  get_utxos_cycles_per_ten_instructions = 10 : nat;
```

Comment:
Should we also update the https://internetcomputer.org/docs/references/bitcoin-how-it-works#api-fees-and-pricing

Reply:
It will be updated separately.